### PR TITLE
Add playsInline option in Video block

### LINKED ISSUE #14422: Video block should have playsinline option
**Is your feature request related to a problem? Please describe.**
The `core/video` block has an  'autoplay' option along with options to hide controls, mute and loop the video. These settings would make this block useful to display gifs in a more efficient video format, for instance. However for autoplay to work on mobile devices the `playsinline` attribute is also required on the `<video>` element. Currently if you enable autoplay and hide controls, the video is not playable on mobile devices. 

**Describe the solution you'd like**
Would be great to have a 'playsinline' option on this block or perhaps add the attribute to the markup automatically if controls are hidden and autoplay is selected. 

**Describe alternatives you've considered**
For the time being I'm adding this with attribute through a `the_content` filter but would be better if it was controllable through the editor.

### PR DESCRIPTION
**Pull request by nfmohit**
## Description
This PR closes #14422 which requests the addition of a `playsInline` option in the `video` block.

## How has this been tested?
This PR has been tested by going through the following steps:

1. Added a Video block to the Gutenberg editor.
2. Made sure the "Playsinline" option shows up in the `InspectorControls`.
3. Made sure turning it on adds the `playsinline` attribute to the `<video>` tag.

## Screenshots 
gutenberg-14422-min

## Types of changes
This PR just mocks the other attributes in the block like "Autoplay", "Muted" and acts accordingly.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/block-library/src/video/edit.js [hunk 1]
```diff
@@ -200,6 +201,11 @@ class VideoEdit extends Component {
 							onChange={ this.toggleAttribute( 'controls' ) }
 							checked={ controls }
 						/>
+						<ToggleControl
+							label={ __( 'Playsinline' ) }
```
Comment (swissspidy) [inline packages/block-library/src/video/edit.js]:
```suggestion
							label={ __( 'Play inline' ) }
```

Comment (nfmohit-wpmudev) [reply packages/block-library/src/video/edit.js]: Thank you so much @swissspidy ❤️ I was quite confused about what to set the label as 😓 Fixed in this [commit](https://github.com/WordPress/gutenberg/pull/14500/commits/c855b7f6cb8dda022809b3dc74a5c8fbb8de5d93).

File: packages/block-library/src/video/edit.js [hunk 2]
```diff
@@ -129,6 +129,7 @@ class VideoEdit extends Component {
 			poster,
 			preload,
 			src,
+			playsinline,
```
Comment (gziolo) [inline packages/block-library/src/video/edit.js]: Can you update this variable/attribute name to be written as `playsInline`? I think it is what React expects and it also reads easier that way.

Comment (nfmohit-wpmudev) [reply packages/block-library/src/video/edit.js]: Thank you so much @gziolo ❤️ Updated [here](https://github.com/WordPress/gutenberg/pull/14500/commits/23cf5beb2b074fb6daa9de96f0de38f86c5a857a).

**Review by swissspidy — APPROVED** [review]

**Comment by gziolo** [comment]
Thanks for yet another contribution 😃

**Comment by nfmohit-wpmudev** [comment]
Thank you so much @gziolo ❤️

**Comment by fienen** [comment]
It looks like this has been merged into Gutenberg master, but I'm not seeing the option in WordPress 5.2.2 still. Has the base version of Gutenberg in WordPress itself not been updated to the version including this patch yet?

**Comment by aduth** [comment]
@fienen Only features from Gutenberg 5.3 and earlier were included in WordPress 5.2.x , which means it's not expected this is present in 5.2.2 (this was included in Gutenberg 5.4).

**Comment by danihazler** [comment]
Sorry, but I still get the following warning when I add a video block to my page.
WordPress 5.7

```
index.js:1 Warning: Invalid DOM property `playsinline`. Did you mean `playsInline`?
    in video
    in figure
```

**Comment by Soean** [comment]
@danihazler Thanks for your feedback, this is an old PR. Can you open a new issue? Which browser are you using?

**Pull request merged**